__tests__/integration.test.js

```diff
@@ -53,6 +53,27 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/users", () => {
+  test("200: returns an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {        
+        for (const user of users) {
```

Comment:
It would be ideal to assert the length of the array of users we get back (to ensure we get them all!)